dash/development/_r_components_generation.py

```diff
@@ -560,6 +584,15 @@ def generate_rpkg(
 written and maintained by the Dash team, is available in\n\
 the dashCoreComponents package. The source for this package\n\
 is on GitHub: plotly/dash-core-components."
+    if package_name in ["dashTable"]:
+        pkg_help_header = "Core Interactive Table Component for Dash"
+        pkg_help_desc = "Dash DataTable is an interactive table component\n\
+designed for viewing, editing, and exploring large datasets. DataTable is\n\
+rendered with standard, semantic HTML <table/> markup, which makes it\n\
+accessible, responsive, and easy to style. This component was written\n\
+from scratch in React.js specifically for the Dash community. Its API\n\
+was designed to be ergonomic and its behavior is completely customizable\n\
+through its properties."
```

Comment:
I recognize that you're just continuing the pattern from above - but these multiline strings with trailing backslashes and no indentation are a nightmare. Let's put them up top in triple quotes like all the other long string constants.

Same goes for `component_helpers` - move it up top so it's appropriate for it to have no indentation.

Reply:
fixed in 8f50b8a17d064ac7da28de182235340e2ea3be43

Reply:
I've made my initial attempt to support `dash-info.yaml` as we discussed, I am 💯certain that there will need to be changes, but felt it was better to start somewhere.

Have not relocated everything to the `.yaml` file, figured we would discuss.